.idea/codeStyles/Project.xml

```diff
@@ -3,7 +3,16 @@
     <JetCodeStyleSettings>
       <option name="PACKAGES_TO_USE_STAR_IMPORTS">
         <value>
-          <package name="kotlinx.android.synthetic" withSubpackages="true" static="false" />
+          <package name="kotlinx.android.synthetic" alias="false" withSubpackages="true" />
```

Comment:
Been seeing churn here lately on projects with codestyle checked in. Haven't dug in but I think it's a change across IDE versions, or something AS hasn't picked up from IDEA yet.